packages/client/src/utils/authUtils.ts

```diff
@@ -54,11 +54,11 @@ export function storeToken(token: string) {
   localStorage.setItem('opencrvs', token)
 }
 
-export function removeToken() {
+export function removeToken(practitionerId: string | null | undefined) {
   const token = getToken()
   if (token) {
     try {
-      authApi.invalidateToken(token)
+      authApi.invalidateToken(token, practitionerId)
```

Comment:
Are the references to `practitionerId` still needed here and in the above `packages/client/src/profile/profileReducer.ts` file? I don't think `authApi.invalidateToken` takes in a second parameter 🤔